src/app/scripts/cac/control/cac-control-sidebar-tab.js

```diff
@@ -0,0 +1,48 @@
+/**
+ *  Controls the sidebar tab switching for the map page
+ *
+ *  Events:
+ *  @event cac:control:sidebartab:shown
+ *  @property {string} tabId The tab id that was selected
+ */
+CAC.Control.SidebarTab = (function ($) {
+
+    'use strict';
+
+    var defaults = {
+        tabWrapperSelector: '.sidebar-tabs'
+    };
+
+    var events = {
```

Comment:
I like this events object and think it should also be implemented for the other controls.